src/biogeophys/TemperatureType.F90

```diff
@@ -1163,15 +1168,18 @@ subroutine InitAccBuffer (this, bounds)
     call init_accum_field (name='T10', units='K', &
          desc='10-day running mean of 2-m temperature', accum_type='runmean', accum_period=-10, &
          subgrid_type='pft', numlev=1,init_value=SHR_CONST_TKFRZ+20._r8)
+    call init_accum_field (name='SOIL10', units='K', &
```

Comment:
Should these be inside an if use_cn statement?

Reply:
This could be within a use_cn statement as it's only used when use_cn is on as well as **onset_thresh_depends_on_veg**. But, **onset_thresh_depends_on_veg** is only local to CNPhenology. So probably makes sense to just be in an use_cn statement.

Reply:
I decided against adding a use_cn if statement. But, we should keep in mind that in many cases these accumulator variables are only used in certain configurations, but are often used all the time. That does allow them to be output to history files. Figuring out when accumulators need to be used would either require complex logic that would cause tendrils of usage beyond the modules that need it -- or a better mechanism to track this sort of thing separately.